var/spack/repos/builtin/packages/basalt/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Basalt(PythonPackage):
```

Comment:
BTW, Python packages are normally prefixed with `py-`. If there's a Basalt package for the C++ library, won't this lead to some conflict?

Reply:
Absolutely. This spack package should be named `py-basalt`.

Reply:
You're right. I will take care of this.